client/src/components/Common/TextInput/index.css

```diff
@@ -0,0 +1,8 @@
+.input-custom{
+  width: 100%;
```

Comment:
minimize width to be more logical than this value, maybe 70% or 65% will be closer to our design. then the other class will control it to change it more for other screens but as a beginning.

Reply:
@Asmaathabet I think 100% width will make it more usable, just put it in a Div and put a padding 
and It will still responsive.

Reply:
Actually, I didn't prefer this way but it's ok if you want to use it in this way